libs/logic-apps-shared/src/utils/src/lib/models/token.ts

```diff
@@ -1,5 +1,15 @@
-import type { TokenType } from '../../editor';
-import type { OpenAPIV2 } from '@microsoft/logic-apps-shared';
+import type { OpenAPIV2 } from '../../';
+
+export const TokenType = {
+  FX: 'fx',
```

Comment:
How does this recognize both tokentype as a type and a const? Won't this cause confusion whoever is importing it? Won't it be better to name it separately

Reply:
This seems to be a pattern we are using throughout LAUX https://github.com/Azure/LogicAppsUX/blob/2a28e353aea28d9b540dc576431dafd4c2a3a53b/libs/designer-ui/src/lib/chatbot/components/conversationItem.ts#L44.

Reply:
@preetriti1 this does suck but it is the best way to do ENUMs in typescript without doing typescript Enums which cause problems with some compilers.

Reply:
I see, I remember the compiler problem with enums..